### PR TITLE
chore: Add attoseconds accessor

### PR DESCRIPTION
**Pull request by hassila**
## Description

Add attoseconds accessor for instant, rename seconds accessor
## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. 

## Minimal checklist:

- [x ] I have performed a self-review of my own code 
- [ ] I have added `DocC` code-level documentation for any public interfaces exported by the package
- [ ] I have added unit and/or integration tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by mr-swifter** [comment]
Will you add init from seconds and attoseconds in the same PR? or?

**Comment by hassila** [comment]
> Will you add init from seconds and attoseconds in the same PR? or?

[f885861](https://github.com/ordo-one/package-datetime/pull/3/commits/f8858614151a1d8874a69c07e8ba1703ac252ceb)

File: Sources/DateTime/InternalUTCClock.swift [hunk 1]
```diff
@@ -30,6 +30,10 @@ public struct InternalUTCClock {
             self._value = _value
         }
 
+        public init(secondsComponent: Int64, attosecondsComponent: Int64) {
```
Comment (mr-swifter) [inline Sources/DateTime/InternalUTCClock.swift:33]: Maybe remove ```Components```? 
```
let instant = InternalUTCClock.Instant(secondsComponent: timestamp.seconds, attosecondsComponent: timestamp.attoseconds)
```

**Review by mr-swifter — APPROVED** [review]

File: Sources/DateTime/InternalUTCClock.swift (hunk 1 above)
Comment (hassila) [reply Sources/DateTime/InternalUTCClock.swift:33]: It is matching `Swift.Duration` in naming convention, thus `Components`...

**Pull request merged**